resources/views/missions/notes/index.blade.php

```diff
@@ -86,14 +71,11 @@
 <div class="mission-comments-form pull-left w-100">
     <form method="post" id="submit-mission-comment">
         <input type="hidden" name="mentions" id="mentions" value="">
-        <textarea class="form-control" id="submit-mission-comment-text" name="text" style="display:none"></textarea>
 
-        <span
+        <textarea
```

Comment:
Looks identical, but using spans instead of text areas causes each line to be put inside a div which is incompatible with markdown